Add files via upload

Anastasiya-Odintsova opened the pull request.

Anastasiya-Odintsova committed:
Add files via upload

Anastasiya-Odintsova committed:
Add files via upload

Nastasia8 commented on 4 модуль.txt:
```diff
@@ -0,0 +1,38 @@
+    print("YES")
+else:
+    print("NO")
+        
```
Кто автор 1 и 2 задачи из модуля 4?

Anastasiya-Odintsova replied:
1 делала я сама. 2 мне подсказал Миша Смирнов

Nastasia8 replied:
Ну, первая тоже похожа на Мишину)

Anastasiya-Odintsova replied:
Если вы сомневаетесь, я могу в субботу объяснить решение

Anastasiya-Odintsova committed:
Add files via upload

Сразу предупрежу, что сам алгоритм я брала у себя из этого же модуля(задача1). Единственное - я спросила у Миши по поводу некоторых последних строчек, именно как правильно это написать. Если вы сомневаетесь, он подтвердит, что в среду я к нему подходила консультироваться и он прекраснейшим образом мне все разъяснил:)

Nastasia8 commented:
Вы бы еще комментарии оставляли по коду (почему применили такую концепцию, в чем увидели зависимость и т.д.)
